deepinv/optim/utils.py

```diff
@@ -57,7 +57,7 @@ def conjugate_gradient(
     def dot(a, b):
         ndim = a[0].ndim if isinstance(a, TensorList) else a.ndim
         dot = (a.conj() * b).sum(
-            dim=tuple(range(1, ndim)), keepdim=False
```

Comment:
I agree with this but this is already the case in the main branch, did you merge from main?

Reply:
I've merged from main and the issue seems to be gone